datamade/how-to
# Upgrades Python, Django, Postgres, and Node dependencies for the Django cookiecutter

## Overview

- closes #278 

## Testing Instructions
- pull down branch and create a new django app
- confirm you can start the django app
- make sure there are no javascript errors in the browser console
